static/sass/_pattern_search-panel.scss

```diff
@@ -2,8 +2,24 @@
   @include vf-u-truncate;
   @include vf-u-text-muted;
 
+
+  .p-search-roles {
+    position: relative;
+
+    &__form {
+      display: flex;
+      justify-content: flex-start;
+    }
+
+    &__group {
+      max-width: 27.6875rem;
+      padding-right: 1.5rem;
+      width: 100%;
+    }
+  }
+
   .p-search-panel {
-    margin-top: 2.3rem;
+    margin-top: -0.5rem;
```

Comment:
This `margin-top: -0.5rem;` and `top: -0.6875rem;` below in line 53 seem like multiple attempts to align search panel with search input.

You should have a clear parent with position relative (`p-search-roles__group` in this case I think). And because the intention is to place panel on the bottom of the input, you could position it from bottom, instead of from top, and likely you only need to compensate for input bottom margin, so `bottom: $input-margin-bottom` should do this.

Reply:
`$input-margin-bottom` doesn't seem to be enough, I need a lot more to position the element below now if I use botto, since it starts where the parent (input) starts.